pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryLocalBeforeBranch.xml

```diff
@@ -356,4 +356,221 @@ public class Example {
 }
 ]]></code>
     </test-code>
-</test-data>
+    <test-code>
+        <description>Simple unnecessary local before throw</description>
+        <expected-problems>1</expected-problems>
+        <code><![CDATA[
+public class Test {
+    void test() {
+        RuntimeException e = new RuntimeException();
+        throw e;
+    }
+}
+        ]]></code>
+    </test-code>
+
+    <!-- Negative case - variable used elsewhere -->
```

Comment:
I think, we don't need these comments that are repeating the test description. it's better to have this in the description (if it is not too long)